### PR TITLE
Updates dropdown menu

### LINKED ISSUE #955: Add newer projects to dropdown and minor UI fixes
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

The dropdown has many bugs like
- Badgeyay icon is stretched for some screemsizes
- Newer projects of fossasia like meilix need to added in the dropdown

**Screenshots of the issue:**

screenshot from 2019-02-05 08-32-16

**Will work on the issue??**

Yes

### PR DESCRIPTION
**Pull request by abhijeetyadav82**
**Changes proposed in this pull request**
- Added the meilix project.
- Fixed the badgeyay icon.

**Screenshots (if appropriate)** 
screenshot from 2019-02-05 09-01-02

**Link to live demo: ** 
http://pr-956-fossasia-loklaksearch.surge.sh
**Closes #955 **

### CODE REVIEW

**Review by shreyanshdwivedi — APPROVED** [review]
LGTM :+1:

**Review by rajvaibhavdubey — APPROVED** [review]
LGTM :+1:

File: src/app/service-box/service-box.component.html [hunk 1]
```diff
@@ -93,12 +93,21 @@
...
 
 						</div>
 						<span>Badgeyay</span>
 					</div>
 				</a>
+				<a href="{{ configUrl.meilix.root }}">
+					<div class="col">
+						<div class="img">
+							<img src="{{ imageUrl.meilix.root }}" width="60" height="60" alt="meilix">
+
```
Comment (simsausaurabh) [inline src/app/service-box/service-box.component.html:106]: Remove extra line

**Review by plxity — APPROVED** [review]

**Review by singhpratyush — CHANGES_REQUESTED** [review]
The text for `meilix generator` overlaps the image on Chrome (haven't checked other browsers) - 

Screenshot from 2019-03-19 08-09-20

**Comment by simsausaurabh** [comment]
Closing due to inactivity. Please follow up with new PR.

**Pull request closed without merging**